crates/interledger-ccp/src/packet.rs

```diff
@@ -117,7 +121,7 @@ impl RouteControlRequest {
         }
 
         PrepareBuilder {
-            destination: CCP_CONTROL_DESTINATION,
+            destination: (*CCP_CONTROL_DESTINATION).clone(),
```

Comment:
The deref can be implicit.
```suggestion
            destination: CCP_CONTROL_DESTINATION.clone(),
```